LICENSE

```diff
@@ -2,7 +2,7 @@ The MIT License (MIT)
 
 Copyright (c) 2016 Felix Wong and contributors
 Copyright (c) 2016 thru 2024, Cloudflare and contributors
-Copyright (c) 2024, Stainless Inc. and contributors
+Copyright (c) 2024 onward, Stainless Inc. and contributors
```

Comment:
did we want to drop this? like in the other PR?

Reply:
Yep, will fix